### ISSUE TITLE
SmolVLA have poor performance

### ISSUE THREAD

**Issue by FanZhang91** [huggingface/lerobot#1370]
### System Info

```Shell
python scripts/train.py --policy.type=smolvla --dataset.repo_id=self_data --batch_size=64 --steps=100000
```

### Information

- [ ] One of the scripts in the examples/ folder of LeRobot
- [ ] My own task or dataset (give details below)

### Reproduction

I use 180 episode data(mobile robot dataset, forklift dataset) to train a model. Using the same data, the pi0 model fit the training data well。But the SmolVLA's prediction results are quite different from GT action.
file:///home/zhangfan/deep_learning/SmolVLA/lerobot/serving_infer/put_next_action.jpg

I change both state and action normalization method to NormalizationMode.MIN_MAX in training phase. 

How to fix this problem?

### Expected behavior

None

**Comment by hahans** [comment]
I think part of the reason may be that the pretraining dataset is entirely focused on the SO - 100 robotic arm, and the overall VLA is relatively small.

**Comment by FanZhang91** [comment]
> I think part of the reason may be that the pretraining dataset is entirely focused on the SO - 100 robotic arm, and the overall VLA is relatively small.

Thanks for your reply。 Is there any way to alleviate this problem?

**Comment by FanZhang91** [comment]
I also use the same data and same model(pi0) to compare lerobot/openpi framework's ability.  The predicted results by the training model(trained by lerobot framework) are also very different from GT actions, but the openpi framework does not have this problem. Is there any potential problem with Lerobot framework itself? @hahans

**Comment by zlw21gxy** [comment]
https://github.com/huggingface/lerobot/issues/1369

**Comment by FanZhang91** [comment]
@zlw21gxy Could you please provide a more detailed description?

**Comment by zlw21gxy** [comment]
@FanZhang91 https://github.com/huggingface/lerobot/issues/1316#issuecomment-3026588850 this is exactly what I use

**Comment by FanZhang91** [comment]
@zlw21gxy Thanks for your reply, but I think my overall pipeline seems functional, and I am not using the Libero dataset but my own data. The comment([1316](https://github.com/huggingface/lerobot/issues/1316#issuecomment-3026588850)) you mentioned above may not be of much help to me.

**Comment by jadechoghari** [comment]
We officially ported LIBERO and have a success rate of 80 to 100 on it:
You can use this checkpoint to evaluate [HuggingFaceVLA/smolvla_libero](https://huggingface.co/HuggingFaceVLA/smolvla_libero) and the doc below https://huggingface.co/docs/lerobot/en/libero
Thank you.

**Comment by jadechoghari** [comment]
This issue has been addressed, so I’ll close it for now.
Please feel free to reopen if needed.

**Comment by Rseelan** [comment]
Having the same issue as well...
